native/src/modules/app/components/App.js

```diff
@@ -16,12 +16,26 @@ import StaticServerProvider from '../../static-server/containers/StaticServerPro
 import I18nProvider from '../../i18n/components/I18nProvider'
 import { NavigationContainer, type LinkingOptions } from '@react-navigation/native'
 import PermissionSnackbarContainer from '../../layout/containers/PermissionSnackbarContainer'
+import { REDIRECT_ROUTE } from 'api-client'
 import NetInfo from '@react-native-community/netinfo'
 
 NetInfo.configure({
   reachabilityUrl: 'https://cms.integreat-app.de/ping'
 })
 
+const linking: LinkingOptions = {
+  prefixes: ['https://', 'integreat://'],
+  // $FlowFixMe redirect is part of available routes
+  config: {
+    screens: {
+      [REDIRECT_ROUTE]: '*'
+    }
+  },
+  getStateFromPath: path => {
+    return { index: 0, routes: [{ name: REDIRECT_ROUTE, params: { url: `https://${path}` } }] }
+  }
+}
+
```

Comment:
Should these changes be in this PR? a little bit confused because this is only about netinfo isnt it?

Reply:
I think you are viewing a single commit/outdated version :thinking: I just see very few changes in App.js.